packages/core/docs/advanced/logging.md

```diff
@@ -1,106 +1,87 @@
 # Logging
 
-In Vue Storefront we're providing all the debugging information, warnings, and errors out of the box so you won't miss anything that happens inside your application. Sometimes you have to pass these data to some third-party monitoring services. You can do this through a custom Logger.
+In Vue Storefront we're providing all the debugging information, warnings, and errors from composables out of the box so you won't miss anything that happens inside your application.  In this document you will learn how to use Logger in your app and how to connect it with external services.
 
+## Using Logger in your app
 
-## Logger interface and configuring
+To make use of Vue Storefront Logger in your application simply import it from the core and use one of 4 available types of messages
 
-Our default implementation uses simple `console` calls, but you are not forced to use it permanently, there is a way to override it or even provide your logger that fit your needs.
-
-To configure and register logger, you need to call `registerLogger` function and pass all of the required functions you need to override.
-
-```ts
-import { registerLogger } from '@vue-storefront/core'
+```js
+import { Logger } from '@vue-storefront/core'
 
-const myLogger = {
-  debug: (message: any, ...args) => console.debug('[VSF][debug]', message, ...args),
-  info: (message: any, ...args) => console.info('[VSF][info]', message, ...args),
-  warn: (message: any, ...args) => console.warn('[VSF][warn]', message, ...args),
-  error: (message: any, ...args) => console.error('[VSF][error]', message, ...args),
-}
+Logger.error('error message')
+Logger.info('info message')
+Logger.warn('warn message')
+Logger.debug('debug message')
+```
 
-const verbosity = 'error'
+## Changing Verbosity
 
-registerLogger(myLogger, verbosity);
-```
+Configuration of Logger happens through `logger` property of `@vue-storefront/nuxt` module. 
 
-Additionally, you can also set the `verbosity` level which tells the app, what you want to log and what communications you want to ignore. By default we have the following verbosity levels:
+You can set the `verbosity` level which tells the app, what you want to log and what communications you want to ignore. By default we have the following verbosity levels:
 
 - `debug` - log everything, including debug calls, information, warnings, and errors (all of the logger functions can be called),
 - `info` - log information, warnings, and errors (debug function calling is skipped),
 - `warn` - log warnings and errors (debug and info functions are skipped),
 - `error` - log only errors (debug, warn and info functions are skipped),
 - `none` - don't log anything.
 
+```js
+['@vue-storefront/nuxt', {
+  logger: {
+    verbosity: 'error' // verbosity
```

Comment:
I think we could get rid of this comment as we have the same on the left

Reply:
yup, overlooked :) Thanks!